What does High Assurance mean? (#231)
openid/OpenID4VC-HAIP: openid4vc-high-assurance-interoperability-profile-1_0.md

This document defines a set of requirements for the existing specifications to enable interoperability among Issuers, Wallets and Verifiers of Credentials where a high level of security and privacy is required. This document is an interoperability profile that can be used by implementations in various contexts, be it a certain industry or a certain regulatory environment.

> The meaning of "high assurance" in the context of this profile is:

Reviewer: There are many concepts in the description here that I do not see in the HAIP profiles. Where are the requirements mentioned about:
- Policies and procedures
- Holder authentication / Holder binding
- Claim authenticity

Reviewer: @tlodderstedt There is currently some confusion in the EU on the link between the HAIP profiles and the LoA High attainment. Could you clarify that while HAIP provides a good basis, it is not sufficient and extra elements that are out of scope would be needed for LoA High

> The meaning of "high assurance" in the context of this profile is:
> 
> * There is a high assurance the claims in a certain credential are valid and bound to the holder (authenticity of the claims). The authenticity is about the policies and procedures to gather and maintain the claims as well as authentication of the holder in the process of the credential issuance and the protection of the authenticity during rest in the wallet and presentation. In scope for this profile is the security of the issuance flow and the protection of the issued credentials as well as ways for verifers to gather data about the issuer. 
> * There is a high assurance the credential presented in a certain transaction is presented by its holder (holder authentication). There are several ways to proof and validate the holder binding. In scope of this profiles is the security of the presentation of key bound credentials. The holder binding of claims-bound can be implemented on top of this functionality. 

Reviewer 2: ```suggestion
The level of "high assurance" aimed to be achieved in the context of this profile includes the following:

* Authenticity of claims: There is strong assurance that the claims within a Credential are valid and bound to the correct Holder. This involves the policies and procedures used to collect and maintain the claims, the authentication of the Holder during issuance, and the protection of claim authenticity both at rest (in the wallet) and during presentation. In scope for this profile are security of the issuance process, protection of issued credentials, and providing mechanisms for the Verifiers to access trustworthy information about the Issuer.
* Holder authentication: There is strong assurance that the Credential is presented by its legitimate Holder in a given transaction. This involves proof of Holder binding, which can be validated through several methods. In scope of this profile are ensuring secure presentation of key-bound credentials, and supporting Claim-based Binding when built on top of this functionality.
```
please capitalize defined terms and use terms as defined

This document defines a set of requirements for the existing specifications to enable interoperability among Issuers, Wallets and Verifiers of Credentials where a high level of security and privacy is required. This document is an interoperability profile that can be used by implementations in various contexts, be it a certain industry or a certain regulatory environment.

The level of "high assurance" aimed to be achieved in the context of this profile includes the following:

> * Authenticity of claims: There is strong assurance that the claims within a Credential are valid and bound to the correct Holder. This involves the policies and procedures used to collect and maintain the claims, the authentication of the Holder during issuance, and the protection of claim authenticity both at rest (in the wallet) and during presentation. In scope for this profile are security of the issuance process, protection of issued credentials, and providing mechanisms for the Verifiers to access trustworthy information about the Issuer.

Reviewer 3: The final sentence of this paragraph is a bit difficult to read. I think it would be easier to read something like:

```suggestion
* Authenticity of claims: There is strong assurance that the claims within a Credential are valid and bound to the correct Holder. This involves the policies and procedures used to collect and maintain the claims, the authentication of the Holder during issuance, and the protection of claim authenticity both at rest (in the wallet) and during presentation. The scope for this profile is: security of the issuance process, protection of issued credentials, and mechanisms for the Verifiers to access trustworthy information about the Issuer.
```

> * Holder authentication: There is strong assurance that the Credential is presented by its legitimate Holder in a given transaction. This involves proof of Holder binding, which can be validated through several methods. In scope of this profile are ensuring secure presentation of key-bound credentials, and supporting Claim-based Binding when built on top of this functionality.

Reviewer 3: Similarly to above, the final sentence is a bit difficult to read (and also uses different wording for the same thing). I think it would be easier to read something like:

```suggestion
* Holder authentication: There is strong assurance that the Credential is presented by its legitimate Holder in a given transaction. This involves proof of Holder binding, which can be validated through several methods. The scope for this profile includes secure presentation of key-bound credentials and supporting Claim-based Binding when built on top of this functionality.
```

> Note: This profile defines how holder authentication can be proved and claim authenticity is protected on a technically level by way of requiring certain protocol and credential format features. Out of scope are concrete holder authentication mechanisms (which ensure the holder can only sign the presentation) and policies and procedures (as this is a technical interop profile and not a policy definition).

Reviewer 3: Grammar:
```suggestion
Note: This profile defines how holder authentication can be proved and claim authenticity is protected on a technical level by way of requiring certain protocol and credential format features. Out of scope are concrete holder authentication mechanisms (which ensure the holder can only sign the presentation) and policies and procedures (as this is a technical interop profile and not a policy definition).
```

> Note: This profile defines how holder authentication can be proved and claim authenticity is protected on a technically level by way of requiring certain protocol and credential format features. Out of scope are concrete holder authentication mechanisms (which ensure the holder can only sign the presentation) and policies and procedures (as this is a technical interop profile and not a policy definition).

Reviewer 3: > which ensure the holder can only sign the presentation

I think this should say:

> which ensure only the holder can sign the presentation

This document defines a set of requirements for the existing specifications to enable interoperability among Issuers, Wallets and Verifiers of Credentials where a high level of security and privacy is required. This document is an interoperability profile that can be used by implementations in various contexts, be it a certain industry or a certain regulatory environment.

> The level of security and privacy defined in this profile includes the following assurance:

Reviewer 4: "defined" feels a bit awkward. Possibly this is slightly less awkward:

```suggestion
This profile aims to achieve a level of security and privacy that includes the following properties:
```

> * Authenticity of claims: There is strong assurance that the claims within a Credential are valid and bound to the correct Holder. This involves the policies and procedures used to collect and maintain the claims, the authentication of the Holder during issuance, and the protection of claim authenticity both at rest (in the wallet) and during presentation. The scope for this profile is: security of the issuance process, protection of issued credentials, and mechanisms for the Verifiers to access trustworthy information about the Issuer.

Reviewer 5: ```suggestion
* Authenticity of claims: There is strong assurance that the claims within a Credential or Presentation are valid and bound to the correct Holder. This involves the policies and procedures used to collect and maintain the claims, the authentication of the Holder during issuance, and the protection of claim authenticity both at rest (in the wallet) and during presentation. The scope for this profile is: security of the issuance process, protection of issued credentials, and mechanisms for the Verifiers to access trustworthy information about the Issuer.
```

* Authenticity of claims: There is strong assurance that the claims within a Credential are valid and bound to the correct Holder. This involves the policies and procedures used to collect and maintain the claims, the authentication of the Holder during issuance, and the protection of claim authenticity both at rest (in the wallet) and during presentation. In scope for this profile are security of the issuance process, protection of issued credentials, and providing mechanisms for the Verifiers to access trustworthy information about the Issuer.
* Holder authentication: There is strong assurance that the Credential is presented by its legitimate Holder in a given transaction. This involves proof of Holder binding, which can be validated through several methods. In scope of this profile are ensuring secure presentation of key-bound credentials, and supporting Claim-based Binding when built on top of this functionality.

> Note: This profile defines how holder authentication can be proved and claim authenticity is protected on a technically level by way of requiring certain protocol and credential format features. Out of scope are concrete holder authentication mechanisms (which ensure the holder can only sign the presentation) and policies and procedures (as this is a technical interop profile and not a policy definition).

Author: thanks, incorporated your suggestion
